### DISCUSSION TITLE
Sharding support in Lettuce client

### DISCUSSION THREAD

**Discussion by akshagr** [redis/lettuce#1862] [Q&A] [answered]
Hi @mp911de 

Is there any way to do Sharding using redisson across multiple standalone servers (non clustered) ?

In ConnectionPoolSupport, while creating generic object pool GenericObjectPoolConfig<T> is used while GenericObjectPoolConfig does not take any type parameter support. Please look into it.

```
public static <T extends StatefulConnection<?, ?>> GenericObjectPool<T> createGenericObjectPool(
            Supplier<T> connectionSupplier, GenericObjectPoolConfig<T> config) {
        return createGenericObjectPool(connectionSupplier, config, true);
    }
```

**Comment by mp911de** [comment, marked as answer]
No, and there is no such support planned. Please use Redis Cluster instead.

**Reply by akshagr** [reply]
What about 2nd question, the ConnectionPoolSupport ?

**Reply by mp911de** [reply]
Care to create a new question for this concern so it can be found by folks having the same issue? Please also attach which dependency versions you're using.
